docs/contributing/contributing.rst

```diff
@@ -396,3 +396,112 @@ Pull Requests
 
   .. _Co-authored-by: https://github.blog/2018-01-29-commit-together-with-co-authors/
   .. _Issue #2816: https://github.com/DataBiosphere/toil/issues/2816
+  .. _toil.lib.retry: https://github.com/DataBiosphere/toil/blob/master/src/toil/lib/retry.py
+
+Adding Retries to a Function
```

Comment:
I'm worried nobody will read this or keep it in sync with the documentation in the code.

Reply:
I'll try to be vigilant in review then.  Hopefully I can keep things up to date.